### DISCUSSION TITLE
build ocaml 4.11 on CentOS 7

### DISCUSSION THREAD

**Discussion by bchill** [ocaml/opam#5359] [Q&A] [answered]
Hello,

I just want to build ocaml 4.1.1 (to use with unison 2.5.1) on CentOS 7.

Is there a simple way to do this without taking on all of opam?

Brian

**Comment by dra27** [comment, marked as answer]
Depends on your definition of simple 🙂 Strictly answer your question: compiling OCaml on CentOS 7 is simple, yes. You'll need gcc and make, then it's just `./configure --prefix /where/to/install/ocaml && make -j && make install`. However, compiling Unison's dependencies will be a lot more (unnecessary) work.

Why do you want to avoid opam? opam can either be downloaded as a binary for CentOS, or compiled from cold (running `make cold` from a git clone will give you an opam binary which you can then install to `/usr/local/bin`, etc.).
